capstone/src/main/java/com/google/sps/servlets/UpdateAssociationServlet.java

```diff
@@ -20,30 +21,41 @@
   public static final String SURVEY_ENTITY_KIND = "Actual";
   public static final String COMMENT_PROPERTY = "text";
 
+  private LanguageServiceClient nlpClient;
+  private DatastoreService datastore;
+
+  public void init() throws ServletException {
+    try {
+      nlpClient = LanguageServiceClient.create();
+      datastore = DatastoreServiceFactory.getDatastoreService();
+    } catch (IOException exception) {
+      System.err.println(exception);
```

Comment:
I believe the error stack trace is automatically printed in Java

Reply:
I think the issue here was that I couldn't just let the error be thrown because of method signatures on the servlet class. Init can't throw the IO exception so you have to catch it in the method at which point I don't believe the error trace is printed because its caught.